src/day_three/mod.rs

```
@@ -0,0 +1,129 @@
+// https://adventofcode.com/2021/day/3
+pub fn execute() {
+    let input = include_str!("input.txt");
+
+    let mut num_bits = 0;
+    let sensor_readings: Vec<u16> = input
+        .lines()
+        .filter_map(|s| match u16::from_str_radix(s, 2) {
+            Ok(i) => {
+                if s.trim().len() > num_bits {
+                    num_bits = s.trim().len()
+                }
+
+                return Option::from(i);
+            }
+            Err(_) => None,
```

Silently ignoring errors doesn't seem like the greatest thing to do. Maybe at the very least, a warning could be printed?